onnxruntime/contrib_ops/cpu/signal/dft.cc

```diff
@@ -39,20 +39,16 @@ ONNX_OPERATOR_KERNEL_EX(
     kMSExperimentalDomain,
     1,
     kCpuExecutionProvider,
-    KernelDefBuilder().MayInplace(0, 0).TypeConstraint("T", BuildKernelDefConstraints<float, double>()),
+    KernelDefBuilder().MayInplace(0, 0).TypeConstraint("T1", BuildKernelDefConstraints<float, double>())
+                                       .TypeConstraint("T2", BuildKernelDefConstraints<int64_t>()),
     STFT);
 
 static bool is_real_valued_signal(const onnxruntime::TensorShape & shape) {
-  // The first dimention is the batch size
```

Comment:
Nit: I liked the comments from before that explained the dimensions

Reply:
This should be explained in the operator schema in detail.